setup.py

```diff
@@ -1,13 +1,21 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
 from __future__ import print_function
 
 import os
+import platform
 import sys
 import fnmatch
 from os.path import join
 from itertools import chain
-from distutils.extension import Extension
+if platform.system() != 'Linux':
+    from distutils.extension import Extension
+    from cx_Freeze import setup, Executable
+else:
+    from setuptools import setup
+    from setuptools.extension import Extension

```

Comment:
I think setuptools is the new norm, at least it is what more exprienced dev I work with on other project use.